src/classes/chat_registry.py

```diff
@@ -76,6 +76,27 @@ async def remove_club(self, chat_id: int):
             chat_id=chat_id, text=f"The club with id #{chat_id} has been removed"
         )
 
+    def get_club(self, chat_id: int) -> Club:
+        """
+        Retrieves a club object based on its chat_id.
+
+        Args:
+            chat_id (int): The chat_id of the club to retrieve.
+
+        Returns:
+            Club: The club object with the specified chat_id, or None if not found.
+        """
+        return self._get_club_by_chat_id(chat_id)
+
+    def update_club(self, club: Club):
+        """
+        Updates the club title.
+
+        Args:
+            club (Club): The update Club instance.
+        """
+        self._update_club(club)
```

Comment:
same, what's the goal?

Reply:
the thought was to add extra logic in public methods in case there will be need. Will get rid of them